test/variables.yaml

```diff
@@ -62,3 +62,15 @@ endpoints:
     endpoint: https://xrootd.physik.uni-bonn.de:1094
     paths:
       wlcg: /cephfs/grid/wlcg
+  se-prague-dcache:
+    desc: PRAGUELCG2 Test dCache
+    type: dCache
+    endpoint: https://dcache.farm.particle.cz
```

Comment:
Hi Petr, is the standard 443 port used for this dCache endpoint?
If not, please specify the correct port

Reply:
Yes, our dCache is configured to listen on standard port 443